models/MenuModel.ts

```diff
@@ -0,0 +1,61 @@
+//Imports
+import Mongoose from 'mongoose';
+import { DbConnection } from "../DbConnection";
+import { IMenuModel } from '../interfaces/IMenuModel';
+
+
+
+//Mongoose connections and object
+let mongooseConnection = DbConnection.mongooseConnection;
+let mongooseObj = DbConnection.mongooseInstance;
+
+//Class for menu model
+class MenuModel {
+    public schema:any;
+    public innerSchema:any;
+    public model:any;
+
+    //constructor initilize the create schema and model
+    public constructor() {
+        this.createSchema();
+        this.createModel();
+    }
+
+    //function to create the schema for Menu
+    public createSchema(): void {
+        this.schema = new Mongoose.Schema(
```

Comment:
Can you add the testing o/p for this

Reply:
I don't understand this request. Can you please elaborate on this further?